doc/manual/manual.tex

```diff
@@ -7381,7 +7381,7 @@ \subsubsection{Postprocessing spherical 3D convection}
 In Figure~\ref{fig:pp} c, d we have subtracted the mean dynamic topography from
 the output field as a postproceesing step outside of Aspect. Since mass is 
 conserved within the Earth, the mean dynamic topography
-should always be zero, however, the outputted values might not fullfill this 
+should always be zero, however, the outputted values might not fulfill this 
```

Comment:
Funny that you needed to replace both fulfil->fulfill and fullfill->fulfill in this one PR :-)